test/visibility/except/exceptModuleItself.good

```diff
@@ -0,0 +1,2 @@
+exceptModuleItself.chpl:5: In function 'main':
+exceptModuleItself.chpl:6: error: Bad identifier in 'except' clause, no known 'Toolbox'
```

Comment:
This error message seems odd to me, because by saying 'use Toolbox' we have to know what 'Toolbox' is.

I think it would be better to have a distinct error for this kind of case where the module name cannot be used in an except/only list if it is an operand(?) of the use-statement. Maybe something like:
```
error: Module 'Toolbox' may not be present in an 'except' clause when it present in the module list of the 'use' statement.
```

Reply:
What about `no known 'Toolbox' defined in module 'Toolbox'`, to clarify that we are looking in the module itself?

Reply:
Does the language allow for anything in that module to share the same identifier as the module?

Reply:
This program compiles and runs:
``` chapel
module M {
  module M {
    proc whatev() {
      writeln("whee");
    }
  }
}

M.M.whatev();
```
I think it's intentional behavior, but it might be that we haven't thoroughly thought it through.  At the very least, it allows foo.chpl to contain:
```chapel 
module foo { ... } // Technically foo.foo
proc somethingElse() { ... }
```

Reply:
Ah, OK. Then your recommended change sounds good, and seems like it could replace the existing "Bad identifier..." text as well (if you want). I'd make sure to add a test for multiple modules in the use-statement as well.

Reply:
We don't allow multiple modules in a use statement with an `except` or `only` list (since we use commas to separate both the modules/enums used and the symbols listed in the limitation clause, you can't have multiple modules/enums with these clauses and then it isn't quite clear whether the limitation should apply to just the last one or apply to all of the listed symbols)